packages/grant-explorer/src/features/discovery/RoundCard.tsx

```diff
@@ -108,7 +108,7 @@ const RoundCard = ({ round }: RoundCardProps) => {
               isValidRoundEndTime={isValidRoundEndTime}
             />
 
-            <RoundBadge strategyName={payoutStrategy.strategyName} />
+            <RoundBadge strategyName={payoutStrategy?.strategyName} />
```

Comment:
@carlbarrdahl do we have empty payout strategy name for old rounds or in other cases? 
does this show an empty badge? in that case shall we completely remove it when we don't have text inside?

Reply:
I'm not sure exactly how this value can be empty for some rounds.

I have updated the RoundBadge to show "Unknown" in these cases. Let me know if you prefer to completely remove it.
https://github.com/gitcoinco/grants-stack/blob/0d773c6dcf97cc2343e77a3cd4481ad2cd7350e0/packages/grant-explorer/src/features/discovery/RoundCard.tsx#L131-L135